training_pipeline/models/model.py

```diff
@@ -163,7 +122,9 @@ def _build_model(num_labels: int) -> tf.keras.Model:
         upsample(64, 3),  # 32x32 -> 64x64
     ]
 
-    inputs = tf.keras.layers.Input(shape=[128, 128, 3])
+    inputs = tf.keras.layers.Input(
+        shape=[128, 128, 3], name=_transformed_name(_IMAGE_KEY)
+    )
```

Comment:
128 should be defined as a meaningful variable.

Reply:
Also, I was thinking maybe the model architecture should live in a separate module and we could then call this script something like `train.py`. Calling it `model.py` is vague IMO. 

WDYT?